recipes/apache.rb

```diff
@@ -15,7 +15,7 @@
             password: node['kibana']['apache']['basic_auth_password'])
   owner node['apache']['user']
   group node['apache']['group']
-  mode 00644
+  mode 0o0644
```

Comment:
My other thought was to switch it back to 0644 which seems fine and more succinct...